src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+
+    <TargetFrameworkNetDesktop>net462</TargetFrameworkNetDesktop>
```

Comment:
We could go with net461 as well, but I figured that we'll likely move WebView2 in here as well and WebView2 requires net462

Reply:
Most will likely ask us for 4.6.1, however given 462 is a serving release of 4.6 I suppose this will be acceptable for most, and more than likely the runtime will be higher on most machines. 

Any reason to not build a netcore package?

---
In reply to: [543456605](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2306#discussion_r543456605) [](ancestors = 543456605)

Reply:
NetCore 3.1 and NetStandard 2.0 targets are defined on lines 5 and 6